src/me/StevenLawson/TotalFreedomMod/Commands/Command_admincmds

```diff
@@ -0,0 +1,22 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
```

Comment:
Thought you were going to make a text message on that you have done the command, but it isnt needed.